boards/arm/stm32g0316_disco/pinmux.c

```diff
@@ -11,10 +11,6 @@
 
 /* pin assignments for STM32G0316-DISCO board */
 static const struct pin_config pinconf[] = {
-#if DT_NODE_HAS_STATUS(DT_NODELABEL(usart1), okay) && CONFIG_SERIAL
-	{STM32_PIN_PA9, STM32G0_PINMUX_FUNC_PA9_USART1_TX},
-	{STM32_PIN_PB7, STM32G0_PINMUX_FUNC_PB7_USART1_RX},
-#endif
 };
 
 static int pinmux_stm32_init(const struct device *port)
```

Comment:
@erwango what should I do for this one ? Looks like it is remapping PA11 to PA9 in **pinmux_stm32_init** when usart1 is used.

Reply:
![image](https://user-images.githubusercontent.com/1338015/95245518-88f39e00-0813-11eb-973f-4ac213040d5a.png)
@erwango @ABOSTM 
If I understand correctly, the G031 (8 pins) has AF **usart1_tx=pa9** and **usart1_rx=pb7**, but only **usart1_cts=pa11** is available on the 8 pins package, so you can remap pa11 to point on pa9 and get access to **usart1_tx=pa9** on pin 5 of the SO8 package. This indeed is necessary if we want the UART to work.

How are we going to handle this very specific case ?

Reply:
@gmarull @erwango @ABOSTM I suggest we fix this one in another PR and just port to the new DT definitions as I did, but this specific remapping might need to be handled in the future.